docs/src/components/Tabs.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { Link } from "gatsby";
+import { Box } from "@kiwicom/orbit-components";
+import useMediaQuery from "@kiwicom/orbit-components/lib/hooks/useMediaQuery";
+import styled, { css } from "styled-components";
+
+export interface TabObject {
+  frontmatter: {
+    title: string;
+  };
+  fields: {
+    slug: string;
+    tabCollection: string | null;
+  };
+}
```

Comment:
It seems that the shape of this interface is affected by how this data is received, could you flatten it?

Reply:
Flattened in Doc layout (which is different than DocLayout 🙊 ).